tests/subsys/bluetooth/mesh/models/app.overlay

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+ / {
+	/* The timer instance to use. */
+	chosen {
+		ncs,dm-timer = &timer2;
+	};
+
+	/* The selected pins will be used by Distance Measurement module for debug purposes. */
```

Comment:
We don't need those pin entries here IMO.

Reply:
@PavelVPV

Reply:
Right, can be removed. Could you please also rename file to `dm.overlay` to reflect  that it belongs to distance measurement module?